shared/whatYouNeed/artik530.md

```diff
@@ -0,0 +1,11 @@
+<img style="float: right;padding-left: 10px;" src="/img/{{ $device.id }}/{{ $device.id }}.jpg" width="25%">
+* A [{{ $device.name }}][device] module mounted on a development board.
+* A 5VDC power supply (100-240V input), if you purchased the development kit you should have one included.
+* At least one wireless communication antenna, also provided as part of the development kit.
```

Comment:
Is the antenna optional, if you're connecting via an Ethernet cable?

Reply:
Yeah, should be optional if wired is used